multiversx_sdk_core/transaction_intent.py

```diff
@@ -0,0 +1,10 @@
+from typing import Optional
+
+
+class TransactionIntent:
+    def __init__(self) -> None:
+        self.sender: str = ""
+        self.receiver: str = ""
+        self.gas_limit: int = 0
+        self.value: Optional[str] = "0"
```

Comment:
Even if the specs say this should be a string, I think here / in Python we can make it an `int`.